publisher/server/publisher.go

```diff
@@ -103,6 +103,8 @@ func (s *Server) publishContainer(container *docker.APIContainers, ttl time.Dura
 			hostAndPort := host + ":" + port
 			if s.IsPublishableApp(containerName) && s.IsPortOpen(hostAndPort) {
 				s.setEtcd(keyPath, hostAndPort, uint64(ttl.Seconds()))
+				dirPath := fmt.Sprintf("/deis/services/%s", appName)
```

Comment:
nitpick: let's move this outside of the inner `for` loop on line 101 so we're not allocating memory every iteration